Automattic/wp-calypso
# Theme Showcase: Fix live demo loading for 3rd-party themes

## Proposed Changes

This PR fixes the issue where the live demo URL for 3rd-party themes would not load correctly in the preview iframe. This is because the preview iframe expects its embedded page to send a `partially-loaded` message. Since the demo URL is externally managed, it doesn't send the `partially-loaded` message, and so the component times out by opening the demo URL in a separate browser tab. 

In order to avoid the weird user experience, this PR proposed to open 3rd-party theme demo sites directly in a new tab, instead of opening the preview modal first.

See the attached screenshots to see how to access a theme's live demo:
| Theme card | Theme detail page |
| --- | --- |
| ![Screenshot 2023-05-03 at 11 31 10 AM](https://user-images.githubusercontent.com/797888/235828091-716f13e3-b05a-4f20-9b4c-aa5221523745.png) | ![Screenshot 2023-05-03 at 11 31 21 AM](https://user-images.githubusercontent.com/797888/235828124-85688780-0086-4f65-94fe-24e387cb1791.png) |


## Testing Instructions



* Head to the Theme Showcase.
* Select any third-party theme, such as Scentina. Third-party themes are labeled with the "Paid" badge. 
* Click on the Live Demo for the theme.
* Ensure that the preview modal opens the demo URL in a new browser tab, instead of opening the preview modal first.
* Also test in the logged-out Theme Showcase.
* Also test with a first-party theme and ensure that it works as before.

## Pre-merge Checklist
